src/XRP/model/payment-flags.ts

```diff
@@ -1,11 +1,13 @@
 /* eslint-disable no-bitwise */
 
 /**
- * Flags used in rippled transactions.
+ * Flags used in payment transactions.
+ *
+ * @note These are only flags which are utilized in Xpring SDK.
+ * For the full list of payment flags, @see https://xrpl.org/payment.html#payment-flags
  *
- * @note These are only flags which are utilized in Xpring SDK. For a complete list of flags, see: https://xrpl.org/transaction-common-fields.html#flags-field.
  */
-class RippledFlags {
+class PaymentFlags {
   static TF_PARTIAL_PAYMENT = 1 << 17
```

Comment:
Any reason we have to encode these constants in bit shift operations rathering than the hex code used in the documentation?